src/detail-page/components/TabJobContent/useFetchJobs.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
It's great that we have extracted the job fetching logic to a separate part of code! 💪